lib/go/README.md

```diff
@@ -132,3 +132,28 @@ if this interval is set to a value too low (for example, 1ms), it might cause
 excessive cpu overhead.
 
 This feature is also only enabled on non-oneway endpoints.
+
+A note about server stop implementations
+========================================
+
+[TSimpleServer.Stop](https://pkg.go.dev/github.com/apache/thrift/lib/go/thrift#TSimpleServer.Stop) will wait for all client connections to be closed after 
+the last received or to be received request to be hanled, so the time spent by stop
```

Comment:
1. typo: `s/hanled/handled/`
2. "after the last received" sounds a bit weird to me.

I would word it like this:

```
When TSimpleServer.Stop is called,
it stops accepting any new client connections,
but will wait for all existing client connections to be closed.
The client connections will only be closed if:

1. The client closes the connection; or
2. The SocketTimeout set on the server is hit.

You can set global variable thrift.ServerStopTimeout early
in your main function to control how long `TSimpleServer.Stop`
would wait:

    // code example here

By default (when it's <=0), it waits indefinitely.
When you set it to a positive value (for example, 5 seconds),
TSimpleServer.Stop will close all the remaining client connections
after the timeout.
Please note that this will cause any ongoing requests to fail,
so in general you should set it to a value that's long enough for
your handlers to handle the requests.
```

Reply:
Emmm, current head will check if `closed` after calling `processor.Process` finish receving and handling a request in `TSimpleServer.processRequests`, if `closed` is set to 1, `inputTransport.Close()` and `outputTransport.Close()` will close the client, so i use the term `last received or to be received request`